packages/components/src/List/types.ts

```diff
@@ -49,3 +49,26 @@ export const itemDimensionKeys = [
   'labelLineHeight',
   'detailFontSize',
 ]
+
+export interface ItemBackgroundColorProps {
+  /**
+   * Replace the normal grey selected and selected + hovered color with brand colors
+   * @default false
+   */
+  brand?: boolean
```

Comment:
It occurs to me that `brand` isn't used anywhere else but `keyColor` is supported by `Link` so it'd be more consistent to use that and create an association the the `theme.colors.key`

```suggestion
  /**
   * Replace the normal uiX color for selected and selected + hovered color with key colors
   * @default false
   */
  keyColor?: boolean
```

Reply:
Resolved in 4b6f7ad